client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -64,32 +68,42 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
             error={touched.email && Boolean(errors.email)}
             value={values.email}
             onChange={handleChange}
+            variant="outlined"
           />
+          <InputLabel htmlFor="password">
+            <Typography className={classes.label}>PASSWORD</Typography>
+          </InputLabel>
           <TextField
             id="password"
-            label={<Typography className={classes.label}>Password</Typography>}
+            placeholder="Password"
             fullWidth
             margin="normal"
             InputLabelProps={{
               shrink: true,
             }}
             InputProps={{
               classes: { input: classes.inputs },
-              endAdornment: <Typography className={classes.forgot}>Forgot?</Typography>,
+              // endAdornment: <Typography className={classes.forgot}>Forgot?</Typography>,
             }}
             type="password"
             autoComplete="current-password"
             helperText={touched.password ? errors.password : ''}
             error={touched.password && Boolean(errors.password)}
             value={values.password}
             onChange={handleChange}
+            variant="outlined"
           />
           <Box textAlign="center">
-            <Button type="submit" size="large" variant="contained" color="primary" className={classes.submit}>
+            <Button type="submit" size="large" variant="contained" color="secondary" className={classes.submit}>
               {isSubmitting ? <CircularProgress style={{ color: 'white' }} /> : 'Login'}
             </Button>
           </Box>
-          <div style={{ height: 95 }} />
+          <Box mt={3} textAlign="center" alignItems="center" justifyContent="center">
+            <Typography style={{ fontWeight: 'bold' }}> Don&apos;t have account? </Typography>
```

Comment:
Let's avoid using inline styling and move this into the `useStyles` file - inline styles are handy for quickly testing something but ideally they are cleaned up before shipping the code 👍🏼